src/main/java/github/scarsz/discordsrv/listeners/PlayerAchievementsListener.java

```diff
@@ -50,17 +54,41 @@ public void onPlayerAchievementAwarded(PlayerAchievementAwardedEvent event) {
 
         // turn "ACHIEVEMENT_NAME" into "Achievement Name"
         String achievementName = PrettyUtil.beautify(event.getAchievement());
+        String channelName = DiscordSRV.getPlugin().getMainTextChannel().getName();
```

Comment:
This would return the name of the channel on Discord, but you later retrieve the in-game channel using this variable.